### PR TITLE
Bump  dev-cpp/random123 and app-text/bibutils

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: app-text/bibutils/bibutils-6.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		-e "s:REPLACE_RANLIB:$(tc-getRANLIB):g" \
+		-e 's:REPLACE_POSTFIX::g' \
+		-e "s:REPLACE_INSTALLDIR:\"${D}/usr/bin\":g" \
+		-e "s:REPLACE_LIBINSTALLDIR:\"${D}/usr/$(get_libdir)\":g" \
+		-e 's:make:$(MAKE):g' \
+		Makefile_start > Makefile \
+		|| die "Failed to set up Makefile"
+
+	cp lib/Makefile.dynamic  lib/Makefile || die
+	cp bin/Makefile.dynamic  bin/Makefile || die
```
Comment (SoapGentoo) [inline app-text/bibutils/bibutils-6.10.ebuild:38]: wird spacing here

File: app-text/bibutils/bibutils-6.10.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	cp test/Makefile.dynamic test/Makefile || die
+}
+
+src_test() {
+	tc-export CC
```
Comment (SoapGentoo) [inline app-text/bibutils/bibutils-6.10.ebuild:43]: do the `tc-export CC` in `src_configure` (this is also the general rule) and drop the `src_test()` then

File: app-text/bibutils/bibutils-6.10.ebuild [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	default
+}
+
+src_install() {
+	dodir /usr/bin
+	emake install
```
Comment (SoapGentoo) [inline app-text/bibutils/bibutils-6.10.ebuild:49]: ...no `DESTDIR`? how can this work?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-text/bibutils/bibutils-6.10.ebuild (hunk 1 above)
Comment (jsmolic) [reply app-text/bibutils/bibutils-6.10.ebuild:38]: I've reworked the ebuild to use the configure script

**Comment by jsmolic** [comment]
Closing in favour of https://github.com/gentoo/gentoo/pull/19481

**Pull request closed without merging**